website/thaliawebsite/settings/settings.py

```diff
@@ -33,10 +33,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
```

Comment:
Isn't `DEBUG` true in this whole file because it's defined here?

Reply:
yes that could be what happens